params/config.go

```diff
@@ -246,6 +250,7 @@ var (
 		RamanujanBlock:      big.NewInt(0),
 		NielsBlock:          big.NewInt(0),
 		MirrorSyncBlock:     big.NewInt(5184000),
+		//BrunoBlock:           big.NewInt(5184000), // TODO: TBD
```

Comment:
can you just assign it as nil, and test whether the new release works on mainnet?